.scalafmt.conf

```diff
@@ -4,3 +4,6 @@ danglingParentheses = true
 binPack.literalsExclude = []
 docstrings = javadoc
 project.git = true
+project.excludeFilters=[
+  tests/unit/src/test/scala/org/scalafmt/tests/CommentsSuite.scala 
```

Comment:
🤦‍♂️ the sooner you give up on vertical alignment the better 😜

Reply:
I'm extracting this in a file.